modules/common/user/user_utils.py

```
@@ -61,24 +61,28 @@ def get_canceled_users(info_db_no: int, origin_table: str):
     return canceled_users
 
 # 구독 모델 판별
+from typing import List, Dict, Tuple
```

Ruff (0.11.9)

Redefinition of unused `List` from line 4

Remove definition: `List`

(F811)